From 1bb39b32d8837b8dfcf17e1fc131edcf9429fdb7 Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Sun, 19 Apr 2020 16:24:14 -0700
Subject: Git 2.18.4

This merges up the security fix from v2.17.5.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
(cherry picked from commit ba6f0905fdb9e65c1ac5fbc79c9a4ef0b59b3e68)
Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 Documentation/RelNotes/2.18.4.txt | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 Documentation/RelNotes/2.18.4.txt

diff --git a/Documentation/RelNotes/2.18.4.txt b/Documentation/RelNotes/2.18.4.txt
new file mode 100644
index 00000000000..e8ef858a00a
--- /dev/null
+++ b/Documentation/RelNotes/2.18.4.txt
@@ -0,0 +1,5 @@
+Git v2.18.4 Release Notes
+=========================
+
+This release merges the security fix that appears in v2.17.5; see
+the release notes for that version for details.
-- 
2.26.1.301.g55bc3eb7cb9

